dist/base-conventions.yml

```diff
@@ -51,3 +51,5 @@ parameters:
       case_insensitive: false
       enforce_no_subject_trailing_period: false
       multiline: false
+      max_body_width: 0
+      max_subject_width: 0
```

Comment:
This has no newline, tsk tsk tsk. What would the code review tool have to say about this?